tree/ntuple/v7/doc/specifications.md

```diff
@@ -501,6 +554,43 @@ The frame hierarchy is as follows
     |---- Column group 2 record frame
     | ...
 
+
+#### Cluster Group Record Frame
+
+The cluster group record frame references the page list envelopes for groups of clusters.
+A cluster group record frame starts with
+
+```
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                      Minimum Entry Number                     +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                          Entry Span                           +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                       Number of clusters                      |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+Followed by the page list envelope link.
+
+The minimum entry number is the minimum first entry of the clusters in the cluster group.
```

Comment:
```suggestion
The minimum entry number is the first entry number of the first cluster in the cluster group.
```

is this correct and maybe less confusing?

Reply:
It's not necessarily correct when cluster sharding is involved (e.g. due to horizontal merging). In this case, clusters are semi-ordered wrt. entry numbers.

Reply:
ok, that is conceptually challenging no matter how we say it then, but maybe:

>To compute the minimum entry number, take first entry number from all clusters in the cluster group, and take minimal among these numbers.